docs/disk-caching/DESIGN.md

```diff
@@ -0,0 +1,46 @@
+# Disk caching design [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
```

Comment:
We follow the TitleCase for doc headers, suggest we do it here as well. This should be `Disk Caching Design`